docker/ubuntu_22.04-dev.dockerfile

```diff
@@ -105,7 +96,7 @@ RUN export PYMOAB_FLAG="-DENABLE_PYMOAB=ON"; \
 # put MOAB on the path
 ENV LD_LIBRARY_PATH $HOME/opt/moab/lib:$LD_LIBRARY_PATH
 ENV LIBRARY_PATH $HOME/opt/moab/lib:$LIBRARY_PATH
-ENV PYTHONPATH=$HOME/opt/moab/lib/python${py_version}/site-packages/
+ENV PYTHONPATH=$HOME/opt/moab/lib/python3.10/site-packages/
```

Comment:
This isn't robust to the introduction of newer versions of python, but I'm not 100% sure what the solution is.